cores/esp8266/FSImpl.h

```diff
@@ -75,6 +76,20 @@ class FSImpl {
     virtual void end() = 0;
     virtual bool format() = 0;
     virtual bool info(FSInfo& info) = 0;
+    // For seamless backward compatibilty, supply default which just copies form the 32bit version
+    virtual bool info64(FSInfo64& info64) {
```

Comment:
This isn't a good idea. If this ever gets called, it will silently do the wrong thing for capacity > 4GB and result in a bug. 
I suggest a pure virtual, or asserting.

Reply:
The alternative here is to replicate the exact functionality in both SPIFFS and LittleFS.  I think it might be better to have the one copy rather than the two.  All other FSes will implement the 64-bit (SD, FTP-FS, SAMBA, etc.) version unless they only support <4GB which seems unlikely.

Reply:
Yeah, fair enough.  A wayward cast could cause odd behavior, and who knows what's living in Arduino-land.  Done.